time[15]    stamp

float64[15] lw_angle
float64[15] rw_angle
float64[15] lw_rvel
float64[15] rw_rvel

int8[15]    lw_pwm
int8[15]    rw_pwm
float64[15] fw_steering

uint16 nb_data